cdap-app-fabric/src/test/java/io/cdap/cdap/internal/app/services/http/AppFabricTestBase.java

```diff
@@ -217,6 +217,7 @@ public abstract class AppFabricTestBase {
   private static MetricStore metricStore;
   private static RemoteClientFactory remoteClientFactory;
   private static LogQueryService logQueryService;
+  public static CConfiguration cConfiguration;
```

Comment:
Don't add this field in this class, the overrided tests might have customized settings in cConf, if you need `cConf` in the unit test, just let call `createBasicCConf` to get it and override your setting